mq/plugins/github_webhooks.py

```diff
@@ -0,0 +1,64 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# Copyright (c) 2017 Mozilla Corporation
+
+import sys
+import os
+import jmespath
+import yaml
+from mozdef_util.utilities.toUTC import toUTC
+
+
+class message(object):
+    def __init__(self):
+        '''
+            Plugin used to fix object type discretions with cloudtrail messages
+        '''
+        self.registration = ['githubeventsqs']
+        self.priority = 10
+
+        with open('mq/plugins/github_mapping.yml', 'r') as f:
```

Comment:
We want to get away from using relative paths here, so we should instead do something like:
```
os.path.join(os.path.dirname(__file__), 'github_mapping.yml')
```

Reply:
Done